src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -135,7 +135,7 @@ public override NodeInputData InputData
                     Name = this.Name,
                     Type = NodeInputTypes.numberInput,
                     Description = this.Description,
-                    Value = System.Convert.ToDouble(Value),
+                    Value = Value,
```

Comment:
@mjkkirschner A bit confused. So except for other cases, this `NumberInputNode` already has `Value` as string?